src/config.ts

```diff
@@ -15,14 +15,45 @@ export interface IndentationMarkerConfiguration {
      * Determines the type of indentation marker.
      */
     markerType?: "fullScope" | "codeOnly"
+
+    /**
+     * Determines the thickness of marker (in pixels).
+     */
+    thickness?: number
+
+    /**
+     * Determines the color of marker.
+     */
+    colors?: {
+        /**
+         * Color of inactive indent markers when using a light theme.
+         */
+        normalLight?: string
+
+        /**
+         * Color of inactive indent markers when using a dark theme.
+         */
+        normalDark?: string
```

Comment:
wdyt about calling this something like `inactiveLight` / `inactiveDark` or just `light` / `dark`? think that's more clear than prefixing with `normal`

Reply:
Good point. `light` / `dark` sound best me